Docs: Deployment guides, introduce AKS guide, strip out azure pieces from GKE guide (#28)
CrowdStrike/falcon-integration-gateway: fig/backends/gcp/README.md

Integration with Google Cloud Platform - Security Command Center.

### Example Configuration file

> [config/config.ini](https://github.com/CrowdStrike/falcon-integration-gateway/blob/main/config/config.ini) is expected to configure Falcon Integration Gateway. Minimal configuration for Azure backend follows.

Reviewer: ```suggestion
[config/config.ini](https://github.com/CrowdStrike/falcon-integration-gateway/blob/main/config/config.ini)  configures Falcon Integration Gateway. Below is a minimal configuration example for GCP:
```
